draft-ietf-modpod-group-processes.md

```diff
@@ -127,58 +127,20 @@ individual's posting rights to IETF mailing lists following a
 by the IESG, often in response to complaints from the community.
 
-This fractured approach to moderation of disruptive participation
-through chairs, list administrators, and moderator teams, combined
-with the IESG-led process of PR-actions, has proven to be less than
-ideal:
-
-- The IETF community has not been able to agree on a common definition
-  of disruptive behavior. Therefore, chairs and list administrators
-  apply individually different criteria when making decisions, and
-  participants have different expectations for when PR-actions are
-  warranted.
-
-- The moderation process that chairs and list administrators need to
-  follow {{?RFC3934}} is slow and cumbersome, which makes it
-  ill-suited to situations that escalate quickly. It also assumes
-  that the originator of disruptive behavior is a misguided
-  participant who can be reasoned with and who will change their
-  ways.
-
-- Chairs and list administrators may only enact moderation actions for
-  their single list, which is ill-suited when a pattern of disruptive
-  behavior spans multiple lists. Also, chairs and list administrators
-  may not be fully aware of disruptive behavior that spans multiple
-  lists, due to not being subscribed to some of the affected.
-
-- PR-actions, which can address disruptive behavior across several
-  lists, are cumbersome and slow, and the community has not been able
-  to agree on a common definition of disruptive behavior. This has
-  led to a situation where PR-actions are rarely used, and when they
-  are used, they are perceived as very heavy-handed.
-
-- For a given mailing list, participants may not feel comfortable
-  reporting disruptive behavior to a chair or list administrator, for
-  various reasons. For mailing lists not associated with working
-  groups, list administrators are not even publicly identified - they
-  can only be contacted through an anonymous alias address. This
-  exacerbates the problem, because participants may not be
-  comfortable reporting disruptive behavior to an anonymous party.
-
-- The IETF offers participation not only through in-person meetings
-  and mailing lists, which are the two channels of participation for
-  which moderation processes are currently defined. IETF business
-  also happens in chat channels, remote meeting participation
-  systems, virtual meetings, wikis, GitHub repositories, and more.
-  How disruptive behavior is moderated in these channels is currently
-  undefined.
+Experience and community input suggests that an evolution of the
+existing processes is necessary.  This memo specifies a flexible moderation
+policy with an eye toward consistency across all IETF communication fora,
+timeliness, fairness, while also addressing transparency, appeals, and
+periodic review of bans.  In particular, moderators are given the broadest
+possible freedom of action to address disruptive behavior in a manner that
+does not disrupt the IETF activities.
```

Comment:
```suggestion
possible freedom of action to address disruptive behavior.
```
Moderation may need to "disrupt IETF activities". If moderators can't take action because of a perceived disruption of IETF activities (whatever that actually means), we gain nothing.